IFTPipeline.jl/src/feature-extraction.jl

```diff
@@ -94,11 +94,12 @@ end
 function extractfeatures_single(;
     input::String, output::String, minarea::Int64, maxarea::Int64, features::Array{String}
 )
-    @info "Loading segmented floes as a binarized image from $input"
-    segmented_floes = BitMatrix(FileIO.load(input))
+    @info "Loading segmented floes from $input"
+    image = FileIO.load(input)
+    labeled_floes = Int64.(reinterpret.(real.(image)))
```

Comment:
Not sure what'd be faster, this or using something like `DelimitedFiles.readdlm(input, Int)`.

Reply:
On reflection this should really be the same function as we have in `tracker.jl`.

Reply:
This PR is updated to include a new pair of functions to load and save the labeled images.